pkg/health/client/client.go

```diff
@@ -345,20 +332,53 @@ func formatNodeStatus(w io.Writer, node *models.NodeStatus, printAll, succinct,
 				formatPathStatus(w, "Secondary", addr, "      ", verbose)
 			}
 		}
+		return true
 	}
+
+	if !nodeIsHealthy(node) {
+		ips := []string{getPrimaryAddressIP(node)}
+		for _, addr := range GetHostSecondaryAddresses(node) {
+			if addr == nil {
+				continue
+			}
+			ips = append(ips, addr.IP)
+		}
+		fmt.Fprintf(w, "  %s%s\t%s\t%s\t%s\n", node.Name,
+			localStr, strings.Join(ips, ","),
+			SummarizePathConnectivityStatusType(GetAllHostAddresses(node)).String(),
+			SummarizePathConnectivityStatusType(GetAllEndpointAddresses(node)).String())
+		return true
+	}
+
+	if allNodes {
+		ips := []string{getPrimaryAddressIP(node)}
+		for _, addr := range GetHostSecondaryAddresses(node) {
+			if addr == nil {
+				continue
+			}
+			ips = append(ips, addr.IP)
+		}
+		fmt.Fprintf(w, "  %s%s\t%s\t%s\t%s\n", node.Name,
+			localStr, strings.Join(ips, ","),
+			SummarizePathConnectivityStatusType(GetAllHostAddresses(node)).String(),
+			SummarizePathConnectivityStatusType(GetAllEndpointAddresses(node)).String())
+		return true
+	}
+
+	return false
 }
 
 // FormatHealthStatusResponse writes a HealthStatusResponse as a string to the
 // writer.
 //
-// 'printAll', if true, causes all nodes to be printed regardless of status
-// 'succinct', if true, causes node health to be output as one line per node
-// 'verbose', if true, overrides 'succinct' and prints all information
+// 'allNodes', if true, causes all nodes to be printed regardless of status
+// 'verbose', if true, prints all information
 // 'maxLines', if nonzero, determines the maximum number of lines to print
-func FormatHealthStatusResponse(w io.Writer, sr *models.HealthStatusResponse, printAll, succinct, verbose bool, maxLines int) {
+func FormatHealthStatusResponse(w io.Writer, sr *models.HealthStatusResponse, allNodes bool, verbose bool, maxLines int) {
```

Comment:
Likely in a separate pr but might be a good idea to just pass in a printOptions struct so we can extend in the future without the need to alter the function ie FormatHealthStatusResponse(io.Writer, HealthStatusResp, PrintOpts)

Reply:
Good idea. I will start looking into this after this PR gets merged.